src/app/components/study-accordion/study-accordion.directive.js

```diff
@@ -43,15 +43,35 @@ angular.module('transmartBaseUi')
 
         node.isLoading = true;
         TreeNodeService.getNodeChildren(node).then(function (result) {
-          console.log('after getNodeChildren')
           node.isLoading = false;
         });
 
       };
 
+      $scope.prev_node = null;
+      $scope.clearMetadata = function (node) {
+        //reset or toggle the flags
+        var isSame = false;
```

Comment:
What's is this for actually? I don't see it has meaningful use because all prev node need to be closed anyway.